network-api/networkapi/templates/pages/research_hub/library_page.html

```diff
@@ -81,68 +83,34 @@ <h1>{{ page.title }}</h1>
                 {% include "fragments/pagination.html" with page=research_detail_pages %}
             </div>
         </div>
-
-        <div id="filter" class="
-            tw-bg-gray-05
-            large:tw-block
-            large:tw-mr-0
-            large:-tw-ml-12
-            large:tw-overflow-y-clip
-            tw-pt-8 large:tw-pt-0
-            tw-px-8 small:tw-px-12 medium:tw-px-16 large:tw-px-12
-            tw-shrink-0
-            large:tw-w-160
-        ">
+        <div id="filter"
+            class=" tw-bg-gray-05 large:tw-block large:tw-mr-0 large:-tw-ml-12 large:tw-overflow-y-clip tw-pt-8 large:tw-pt-0 tw-px-8 small:tw-px-12 medium:tw-px-16 large:tw-px-12 tw-shrink-0 large:tw-w-160 ">
```

Comment:
Is this change due to linting complaint? The original format looks more readable :(